docs/Learning-Environment-Design-Agents.md

```diff
@@ -200,29 +200,53 @@ used in your normalization formula.
+CNNs, see [this guide](http://cs231n.github.io/convolutional-networks/). You 
+can use visual observations along side vector observations.
 
-Agents using camera images can capture state of arbitrary complexity and are
-useful when the state is difficult to describe numerically. However, they are
-also typically less efficient and slower to train, and sometimes don't succeed
-at all.  
+Agents using visual observations can capture state of arbitrary complexity and 
+are useful when the state is difficult to describe numerically. However, they 
+are also typically less efficient and slower to train, and sometimes don't 
+succeed at all.
 
-To add a visual observation to an Agent, click on the `Add Camera` button in the
-Agent inspector. Then drag the camera you want to add to the `Camera` field. You
-can have more than one camera attached to an Agent.
+To add a visual observation to an Agent, either click on the `Add Camera` or 
+`Add RenderTexture` button in the Agent inspector. Then drag the camera or 
+render texture you want to add to the `Camera` or `RenderTexture` field. 
+You can have more than one camera or render texture and even use a combination 
+of both attached to an Agent.
 
 ![Agent Camera](images/visual-observation.png)
 
+or
+
+![Agent RenderTexture](images/visual-observation-rendertexture.png)
+
 In addition, make sure that the Agent's Brain expects a visual observation. In
 the Brain inspector, under **Brain Parameters** > **Visual Observations**,
-specify the number of Cameras the Agent is using for its visual observations.
+specify the number of Resolutions the Agent is using for its visual observations.
 For each visual observation, set the width and height of the image (in pixels)
 and whether or not the observation is color or grayscale (when `Black And White`
-is checked).
+is checked). If a combination of `Cameras` and `RenderTextures` is used, all 
+cameras are captured first, then all render textures will be added, both in the
+order they appear in the editor. With 2 cameras and 1 render texture of say twice 
+the size, 3 **Visual Observations** have to be added to the **Brain Parameters**,
+2 with the base resolution and 1 with doubled dimensions. Then the two cameras
+and the render texture, have to be added under `Cameras` and `RenderTextures` on
+the agent.
+
+![Agent Camera and RenderTexture combination](images/visual-observation-combination.png)
+
+RenderTexture observations will throw an `Exception` if the width/height, doesn't 
+match the resolution specified under **Brain Parameters** > **Visual Observations**.
+
+When using `RenderTexture` visual observations a handy feature for debugging, is 
+adding a `Canvas` with a `RawImage` with it's texture set to the `RenderTexture`, 
+in order to view the agent observation. The [GridWorld example](Learning-Environment-Examples.md#gridworld) 
+GridWorldRenderTexture scene uses a RenderTexure for debugging and observation.
```

Comment:
There is no explanation on how to update the renderTexture. I think there needs to be a note and maybe a screenshot on how to drag the renderTexture into the field of the camera. <-- This is not obvious.

Reply:
Added screenshot and reworded this paragraph